src/libs/actions/IOU.js

```diff
@@ -311,6 +311,27 @@ function getReceiptError(receipt, filename, isScanRequest = true) {
         : ErrorUtils.getMicroSecondOnyxErrorObject({error: CONST.IOU.RECEIPT_ERROR, source: receipt.source, filename});
 }
 
+/**
+ * @param {Object} [policy]
```

Comment:
can we add some comment explaining this function? i think it'd be nice to explain the why.

Reply:
@tienifr could we add a comment to explain the why, please? ty!